perfrunbook/utilities/install_perfrunbook_dependencies.sh

```diff
@@ -7,6 +7,9 @@ install_al2_dependencies () {
   echo "------ INSTALLING HIGH LEVEL PERFORMANCE TOOLS ------"
   yum install -y -q sysstat dstat htop hwloc tcpdump
 
+  # Update sysstat to the latest version
+  yum update -y -q sysstat
```

Comment:
Is this necessary?  `yum install` should pick up the latest version the first time correct?